plugins/source/aws/client/spec.go

```diff
@@ -42,6 +42,7 @@ type Spec struct {
 	PartitionID               string    `json:"custom_endpoint_partition_id,omitempty"`
 	SigningRegion             string    `json:"custom_endpoint_signing_region,omitempty"`
 	InitializationConcurrency int       `json:"initialization_concurrency"`
+	SkipNonFreeAPIs           bool      `json:"use_non_free_apis"`
```

Comment:
```suggestion
	UseNonFreeAPIs            bool      `json:"use_non_free_apis"`
```

Reply:
```suggestion
	IncludePaidAPIs           bool      `json:"include_paid_apis"`
```
I think this is today's consensus :)